libs/utils/src/tallies.ts

```diff
@@ -0,0 +1,35 @@
+import { strict as assert } from 'assert'
+import {
+  ContestOptionTally,
+  ContestTally,
+  Dictionary,
+} from '@votingworks/types'
+
+export function combineContestTallies(
+  firstTally: ContestTally,
+  secondTally: ContestTally
+): ContestTally {
+  assert(firstTally.contest.id === secondTally.contest.id)
```

Comment:
`assert.equal` will give better error messages:
```suggestion
  assert.equal(firstTally.contest.id, secondTally.contest.id)
```